documents/WCA Bylaws.md

```diff
@@ -6,7 +6,7 @@
 <div style="margin: 200px auto; font-size: 1.8em;">
 # World Cube Association
 </div>
-<img src="../assets/WCAlogo_notext.svg" style="width: 60%; margin: auto;">
+<img src="{logo}" style="width: 60%; margin: auto;">
```

Comment:
You can change this to a standard Markdown image reference `![My image](/path/to/image)` and add the styling properties via curly braces

Reply:
I just kept it as it was done originally, but sure, I can change this to markdown

Reply:
Fair point, but in this case it is obvious that the img tag is exactly equivalent to a Markdown image ![]() reference